test/VoiceProcessorTestApp/.editorconfig

```diff
@@ -0,0 +1,15 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
+
+root = true
+
+[*]
+
+indent_style = space
+indent_size = 4
```

Comment:
don't we do 2 spaces for js/ts?

Reply:
yeah, but the problem with using that with react native is it doesn't mix with the java codestyle conventions which require 4 to pass. I'll try to find a way to support both.

Reply:
ok, if its too much its fine as is.

Reply:
I removed editorconfigs and just let prettier, xcode and android studio do their own formatting. Works better this way.